CHANGELOG.rst

```diff
@@ -7,7 +7,7 @@ This document describes changes between each past release.
 8.1.0 (unreleased)
 ==================
 
-- Nothing changed yet.
+- Allow reading from batch contexts using the ``parse_results`` method. (#146)
```

Comment:
I would suggest to be more human friendly with something like:

`Allow reading responses from batch requests with the ``results()`` method. (#146)`